packages/composables/src/useCart.ts

```diff
@@ -278,6 +279,10 @@ export function useCartFunction(): UseCartReturn {
     return errors;
   };
 
+  const getItem = (id: string) => {
```

Comment:
I'd skip it from modifying `useCart` api and just add this inside error resolver, we could add this here if there would be more broadly needed

Reply:
done